src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/eventDecoder.hpp

```diff
@@ -55,6 +52,16 @@ class EventDecoder final : public AbstractHandler<std::shared_ptr<EventContext>>
             {
                 data->resourceType = ResourceType::CVE;
             }
+            else if (Utils::startsWith(data->resource.at("resource"), "FEED-GLOBAL"))
+            {
+                data->resourceType = ResourceType::VENDOR_MAP;
+            }
+            else if (Utils::startsWith(data->resource.at("resource"), "TOSID-") ||
+                     Utils::startsWith(data->resource.at("resource"), "OSCPE-GLOBAL"))
```

Comment:
The TOSID- events will be deprecated with the next CTI content release:
```suggestion
            else if (Utils::startsWith(data->resource.at("resource"), "OSCPE-GLOBAL"))
```

Reply:
Done!